gateway-messages/tests/versioning/v13.rs

```diff
@@ -0,0 +1,123 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! The tests in this module check that the serialized form of messages from MGS
+//! protocol version 12 have not changed.
+//!
+//! If a test in this module fails, _do not change the test_! This means you
+//! have changed, deleted, or reordered an existing message type or enum
+//! variant, and you should revert that change. This will remain true until we
+//! bump the `version::MIN` to a value higher than 11, at which point these
+//! tests can be removed as we will stop supporting v11.
```

Comment:
```suggestion
//! bump the `version::MIN` to a value higher than 13, at which point these
//! tests can be removed as we will stop supporting v13.
```

(or feel free to reword this to omit the number, so the next person that grabs this comment block for a new file doesn't have to update it)

Reply:
Reworded for future selves.